src/helpers/discord_utils.ts

```diff
@@ -217,75 +211,115 @@ export async function sendEndRoundMessage(messageContext: MessageContext,
+    for (const gameOptionConflictCheck of gameOptionConflictCheckMap) {
+        const doesConflict = gameOptionConflictCheck.conflictCheck();
+        if (doesConflict) {
+            for (const option of ConflictingGameOptions[gameOptionConflictCheck.gameOption]) {
+                if (optionStrings[option] !== null) {
+                    optionStrings[option] = `${strikethrough(optionStrings[option])} (${CONFLICT_WARNING_EMOJI}\`${PREFIX}${GameOptionCommand[gameOptionConflictCheck.gameOption]}\` ${CONFLICT})`;
+                } else {
+                    optionStrings[option] = "";
+                }
+            }
+        }
+    }
+
+    // Surround changed option with code block, underline all other "set" options
+    // Unset options are set to NOT_SET_OPTION later
     for (const gameOption of Object.keys(optionStrings)) {
         const gameOptionString = optionStrings[gameOption];
         if (!gameOptionString) continue;
-        optionStrings[gameOption] = (updatedOption && updatedOption.option) === gameOption ? bold(gameOptionString) : codeLine(gameOptionString);
+        if (updatedOption && updatedOption.option === gameOption) {
+            optionStrings[gameOption] = underline(gameOptionString);
+        }
     }
 
-    const goalMessage = `First one to ${optionStrings[GameOption.GOAL]} points wins.`;
-    const guessTimeoutMessage = ` in less than ${optionStrings[GameOption.TIMER]} seconds`;
-    const shuffleMessage = `Songs will be shuffled in ${optionStrings[GameOption.SHUFFLE_TYPE]} order. `;
+    // Options excluded from embed fields since they are of higher importance (shown above them as part of the embed description)
+    const priorityOptions = PriorityGameOption
+        .map((option) => `${bold(PREFIX + GameOptionCommand[option])}: ${optionStrings[option] || NOT_SET_OPTION}`)
+        .join("\n");
+
+    const fieldOptions = Object.keys(GameOptionCommand).filter((option) => !PriorityGameOption.includes(option as GameOption));
+    const firstHalfOptions = fieldOptions.slice(0, fieldOptions.length / 2);
+    const secondHalfOptions = fieldOptions.slice(fieldOptions.length / 2);
+    const ZERO_WIDTH_SPACE = "​";
+    const fields = [
+        {
+            name: ZERO_WIDTH_SPACE,
+            value: firstHalfOptions.map((option) => `${bold(PREFIX + GameOptionCommand[option])}: ${optionStrings[option] || NOT_SET_OPTION}`).join("\n"),
+            inline: true,
+        },
+        {
+            name: ZERO_WIDTH_SPACE,
+            value: secondHalfOptions.map((option) => `${bold(PREFIX + GameOptionCommand[option])}: ${optionStrings[option] || NOT_SET_OPTION}`).join("\n"),
+            inline: true,
+        },
+    ];
 
-    if (updatedOption && updatedOption.reset) {
-        footerText = `Looking for information on how to use this command? Check out '${process.env.BOT_PREFIX}help [command]' to learn more`;
+    if (!footerText && Math.random() < 0.25) {
```

Comment:
It should always show this text if an option was reset, incase a person doesn't know how to use the command and is resetting accidentally